vdpau_video.c:927:1: error: conflicting types for 'vdpau_CreateSurfaceFromV4L2Buf'
  927 | vdpau_CreateSurfaceFromV4L2Buf(
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from vdpau_video.c:22:
vdpau_video.h:274:1: note: previous declaration of 'vdpau_CreateSurfaceFromV4L2Buf' was here
  274 | vdpau_CreateSurfaceFromV4L2Buf(
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
diff --git src/vdpau_video.c src/vdpau_video.c
index 41d4573..937de19 100644
--- a/src/vdpau_video.c
+++ b/src/vdpau_video.c
@@ -927,8 +927,10 @@ VAStatus
 vdpau_CreateSurfaceFromV4L2Buf(
     VADriverContextP    ctx,
     int                 v4l2_fd,
+/*
     struct v4l2_format *v4l2_fmt,
     struct v4l2_buffer *v4l2_buf,
+*/
     VASurfaceID        *surface
 )
 {
diff --git src/vdpau_video.h src/vdpau_video.h
index 667a6d7..9ba654f 100644
--- a/src/vdpau_video.h
+++ b/src/vdpau_video.h
@@ -274,8 +274,10 @@ VAStatus
 vdpau_CreateSurfaceFromV4L2Buf(
     VADriverContextP    ctx,
     int                 v4l2_fd,
+/*
     struct v4l2_format *v4l2_fmt,
     struct v4l2_buffer *v4l2_buf,
+*/
     VASurfaceID        *surface
 ) attribute_hidden;
 
